modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm

```diff
@@ -360,7 +360,6 @@
 	w_class = WEIGHT_CLASS_NORMAL
 
 /obj/item/storage/box/gunset/sarge/PopulateContents()
-	new /obj/item/trim_token/security_sergeant(src)
```

Comment:
remove the whole gunset box since it only applies to sarges anyway.

Reply:
I figured I'd leave it in if anyone needed it for stuff, but I suppose I can do that too.

Reply:
you removed the only reasonable way to obtain the box so it's not really needed.